netebpfext/net_ebpf_ext_bind.c

```diff
@@ -81,6 +82,7 @@ ebpf_extension_data_t _net_ebpf_extension_bind_hook_provider_data = {
 NPI_MODULEID DECLSPEC_SELECTANY _ebpf_bind_hook_provider_moduleid = {sizeof(NPI_MODULEID), MIT_GUID, {0}};
 
 static net_ebpf_extension_hook_provider_t* _ebpf_bind_hook_provider_context = NULL;
+static bool _ebpf_bind_hook_info_provider_registered = false;
```

Comment:
Ok for now but code would be simpler to just use _ebpf_bind_hook_provider_context as NULL if not registered and non-NULL if registered, if possible.

Reply:
Fixed.